src/frontend/index.tsx

```diff
@@ -87,11 +86,26 @@ document.addEventListener('DOMContentLoaded', async (event) => {
     cache,
   );
 
+  console.log('into frontend index, about to find <App />');
+
+  let App: () => JSX.Element;
+  if (appData.frontend === 'LTI') {
+    const { LTIRoutes } = await import('./components/LTIRoutes');
```

Comment:
if the routes component return a default we could maybe do something like this : 

```
try {
    const App = await import(`./components/${appData.frontend}Routes`);
} catch(error) {
    throw new Error(
      `${appData.frontend} is not an expected value for appData.frontend`,
    );
}
```

Reply:
I don't see a reason not to do this. I just used the same export name instead of a default export, which does not change the resulting code too much.